Assets/Shopify/UIToolkit/Shops/Generic/Classes/MultiProductListItem.cs

```diff
@@ -27,13 +27,15 @@ public class MultiProductListItem : MonoBehaviour {
 
             var images = (List<Shopify.Unity.Image>)product.images();
             if (images.Count > 0) {
+                if (ImageLoader.LoadingInProgress) {
+                    ImageLoader.CancelPreviousLoad();
```

Comment:
oh i see... so if no new images are returned then we need to stop some loading loop?